php/Terminus/Commands/SitesCommand.php

```diff
@@ -453,7 +453,7 @@ public function massUpdate($args, $assoc_args) {
           }
           // Backup the DB so the client can restore if something goes wrong.
```

Comment:
@TeslaDethray: Thank you for the quick fix.  The comment says backup the db but doesn't 'all' element mean code, db and files?